test/powershell/Modules/Microsoft.PowerShell.Core/Enter-PSHostProcess.Tests.ps1

```diff
@@ -133,7 +133,23 @@ Describe "Enter-PSHostProcess tests" -Tag Feature {
                 $rs.Open()
                 $ps = [powershell]::Create()
                 $ps.Runspace = $rs
-                $ps.AddScript('$pid').Invoke() | Should -Be $pwshId
+                $ps.AddScript('$pid')
+
+                [int]$retry = 0
```

Comment:
Can we use `Wait-UntilTrue` from HelpersCommon.psm1?

Reply:
If not, maybe add a retry function on HelpersCommon.

Reply:
I worked out a `Invoke-ScriptBlockWithRetry` function, but it doesn't save much and makes the code harder to understand since I need to pass two types of information back.